### PR TITLE
fixed job information not being stored when creating submission

### PR DESCRIPTION
**Pull request by vsolovei-smartling**
job information was created/updated if submission already exists

### CODE REVIEW

File: inc/autoload.php [hunk 1]
```diff
@@ -26,5 +26,11 @@
...
-
-
+$kernel = \AspectMock\Kernel::getInstance();
+$kernel->init([
+    'cacheDir' => __DIR__ . '/../aopcache',
+    'debug' => true,
+    'excludePaths' => [__DIR__ . '/third-party'],
+    'includePaths' => [__DIR__],
+]);
+$kernel->loadFile(__DIR__ . '/Smartling/Submissions/SubmissionEntity.php');
```
Comment (PavelLoparev) [inline inc/autoload.php:36]: Is this for tests only?

File: inc/Smartling/Services/ContentRelationsDiscoveryService.php [hunk 2]
```diff
@@ -273,7 +271,7 @@ public function createSubmissionsHandler($data = ''): void
 
                     // trigger generation of fileUri
                     $submission->getFileUri();
-                    $this->submissionManager->storeEntity($submission);
+                    $this->storeWithJobInfo($submission, $jobInfo);
```
Comment (PavelLoparev) [inline inc/Smartling/Services/ContentRelationsDiscoveryService.php:274]: Do we have other places where we store submission but do not pass job info?

File: inc/autoload.php (hunk 1 above)
Comment (vsolovei-smartling) [reply inc/autoload.php]: Yes, but I've decided against using AspectMock, slows tests too much

File: inc/Smartling/Services/ContentRelationsDiscoveryService.php (hunk 2 above)
Comment (vsolovei-smartling) [reply inc/Smartling/Services/ContentRelationsDiscoveryService.php:274]: I don't think so

**Review by PavelLoparev — APPROVED** [review]

File: readme.txt [hunk 3]
```diff
@@ -60,6 +60,9 @@ Additional information on the Smartling Connector for WordPress can be found [he
 3. Track translation status within WordPress from the Submissions Board. View overall progress of submitted translation requests as well as resend updated content.
 
 == Changelog ==
+= 2.4.3 =
+* Fixed an issue where job links were not displayed when first sending content for translation
```
Comment (dimitrystd) [inline readme.txt:64]:
```suggestion
* Fixed an issue when job links were not displayed on `Translation Progress` after submitting content for translation for the first time
```

**Review by dimitrystd — APPROVED** [review]
Greate that you found a solution!

**Pull request merged**